<launch>
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>
  <node name="sound_play_gui" pkg="sound_play_gui" type="sound_play_controller.py"/>
</launch>